skbio/metadata/_interval.py

```diff
@@ -264,11 +264,15 @@ def _check_bounds(self, bounds):
         '''input `bounds` must be sorted.'''
         upper_bound = self._interval_metadata.upper_bound
         lower_bound = self._interval_metadata.lower_bound
-        if bounds[-1][-1] > upper_bound or bounds[0][0] < lower_bound:
+        if upper_bound is not None and bounds[-1][-1] > upper_bound:
             raise ValueError('Cannot set `bounds` (%r) with coordinate '
-                             'larger than upper bound (%r) or smaller than '
-                             'lower bound (%r).' %
-                             (bounds, upper_bound, lower_bound))
+                             'larger than upper bound (%r)' %
+                             (bounds, upper_bound))
+
+        if bounds[0][0] < lower_bound:
```

Comment:
lower_bound is essentially a stub currently, leaving it there for future use cases. would prefer to leave it as it is.